modules/oe_paragraphs_demo/config/install/user.role.editor.yml

```diff
@@ -0,0 +1,23 @@
+langcode: en
```

Comment:
I don't see what is the issue with having a testing role with permissions. Here we need several permissions rather than just a couple. This way we keep the tests clean from listing several lines of permissions to achieve the goal.